### PR TITLE
Mailbox instructions link

### PR DESCRIPTION
**Pull request by ahwnsals1234**
Added link of the instruction videos to the mailbox instructions referencing #1815

### CODE REVIEW

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 1]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                    <Typography sx={{ fontSize: '0.8rem' }}>
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                          fontSize: '1.1rem',
+                          fontWeight: 'bold',
+                        }}
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:409]:
```suggestion
                        style={{
                          borderBottom: '1px solid currentColor',
                          textDecoration: 'none',
                          color: gordonColors.primary.blueShades.A700,
                          fontSize: '1.1rem',
                          fontWeight: 'bold',
                        }}
```

Is there specific reason you don't have your styling inside of a `css` file?

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 2]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                        href="https://m.youtube.com/shorts/FxE5PPS94sc"
+                        underline="always"
+                        target="_blank"
+                        rel="noopener"
+                      >
+                        Salsbury Mailbox
+                      </Link>
+                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:417]:
```suggestion
                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```

Typography definitely can have its style in css

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 3]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                      <br />
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                          fontSize: '1.1rem',
+                          fontWeight: 'bold',
+                        }}
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:441]: ditto
```suggestion
                        style={{
                          borderBottom: '1px solid currentColor',
                          textDecoration: 'none',
                          color: gordonColors.primary.blueShades.A700,
                          fontSize: '1.1rem',
                          fontWeight: 'bold',
                        }}
```

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 4]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                        href="https://m.youtube.com/shorts/47402r3FqSs"
+                        underline="always"
+                        target="_blank"
+                        rel="noopener"
+                      >
+                        Dial and Pointer Mailbox
+                      </Link>
+                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:449]:
```suggestion
                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```

ditto

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 5]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                      <br />
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                          fontSize: '1.1rem',
+                          fontWeight: 'bold',
+                        }}
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:476]:
```suggestion
                        style={{
                          borderBottom: '1px solid currentColor',
                          textDecoration: 'none',
                          color: gordonColors.primary.blueShades.A700,
                          fontSize: '1.1rem',
                          fontWeight: 'bold',
                        }}
```

ditto

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 6]
```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
+                        href="https://m.youtube.com/shorts/0VuTFs1Iwnw"
+                        underline="always"
+                        target="_blank"
+                        rel="noopener"
+                      >
+                        Double Dial Mailbox
+                      </Link>
+                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:484]:
```suggestion
                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```
ditto

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 7]
```diff
@@ -55,6 +59,10 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
     AuthGroup.SensitiveInfoView,
     AuthGroup.AcademicInfoView,
   );
+  const [isJoinDialogOpen, setIsJoinDialogOpen] = useState(false);
+  const onClose = () => {
+    setIsJoinDialogOpen(false);
+  };
```
Comment (amos-cha) [inline src/components/Profile/components/PersonalInfoList/index.jsx:65]:
```suggestion
  const onClose = () => {
    setIsJoinDialogOpen(false);
  };
```

`onClose` is called only once. You should probably just set the state from the location directly

**Review by amos-cha — CHANGES_REQUESTED** [review]

**Comment by ahwnsals1234** [comment]
@amos-cha Made all the fixes according to the review, would be appreciated if you can take another look.

**Review by amos-cha — APPROVED** [review]

**Pull request merged**